src/test/regress/spec/isolation_update_node.spec

```diff
@@ -103,43 +104,45 @@ step "s2-execute-prepared" {
     EXECUTE foo;
 }
 
-step "s2-verify-metadata"
+step "s2-abort"
 {
-    SELECT nodeid, groupid, nodename, nodeport FROM pg_dist_node ORDER BY nodeid;
-    SELECT master_run_on_worker(
-        ARRAY['localhost'], ARRAY[57638],
-        ARRAY['SELECT jsonb_agg(ROW(nodeid, groupid, nodename, nodeport) ORDER BY nodeid) FROM  pg_dist_node'],
-        false);
+	ABORT;
 }
 
-step "s2-start-metadata-sync-node-2"
+session "s3"
+
+step "s3-update-node-1-back"
 {
-    SELECT start_metadata_sync_to_node('localhost', 57638);
+    SELECT 1 FROM master_update_node(
+        (select nodeid from pg_dist_node where nodeport = 58637),
+        'localhost',
+        57637);
 }
 
-step "s2-drop-table" {
-    DROP TABLE test;
+step "s3-update-node-2-back"
+{
+    SELECT 1 FROM master_update_node(
+        (select nodeid from pg_dist_node where nodeport = 58638),
+        'localhost',
+        57638);
 }
 
-step "s2-abort"
+
+// since we update the nodes to unexistent nodes we break metadata, so here we fix it manually
+step "s3-manually-fix-metadata"
 {
-	ABORT;
+    UPDATE pg_dist_node SET metadatasynced = 't' WHERE nodeport = 57637;
+    UPDATE pg_dist_node SET metadatasynced = 't' WHERE nodeport = 57638;
```

Comment:
Can we move `SELECT start_metadata_sync_to_node('localhost', 57638);` here? So, we change in the coordinator an immediately sync the workers back? That feels safer / easier to follow compared having it on the tear down